### PR TITLE
support ledger app v1.1

### PR DESCRIPTION
**Pull request by kroggen**
The new version of the Ledger App can request to re-send the transaction from the first chunk

### CODE REVIEW

File: packages/ledger-hw-app-aergo/src/index.ts [hunk 1]
```diff
@@ -169,9 +170,14 @@ export default class LedgerAppAergo {
             const response = await wrapRetryStillInCall(() =>
                 this.transport.send(CLA, INS.SIGN_TX, mode, 0x00, dataChunk)
             );
-            if (response.length && response.toString('hex') !== TX_REQUEST_NEXT_PART) {
-                const [hash, signature] = chunkBy(response, [32, response.length - 32 - 2]);
-                return [hash, signature];
+            if (response.length) {
+                var hex_response = response.toString('hex');
```
Comment (graup) [inline packages/ledger-hw-app-aergo/src/index.ts]:
```suggestion
                const hexResponse = response.toString('hex');
```

Comment (kroggen) [reply packages/ledger-hw-app-aergo/src/index.ts]: How did you made this?
If you update the variable name in the 2 occurrences below I can add in a single commit

Comment (kroggen) [reply packages/ledger-hw-app-aergo/src/index.ts]: *constant

File: packages/ledger-hw-app-aergo/src/index.ts [hunk 2]
```diff
@@ -169,9 +170,14 @@ export default class LedgerAppAergo {
...
+                if (hex_response === TX_REQUEST_FIRST_PART) {
+                    offset = -chunkSize;
+                } else if (hex_response !== TX_REQUEST_NEXT_PART) {
```
Comment (graup) [inline packages/ledger-hw-app-aergo/src/index.ts]:
```suggestion
            if (response.length) {
                const hexResponse = response.toString('hex');
                if (hexResponse === TX_REQUEST_FIRST_PART) {
                    offset = -chunkSize;
                } else if (hexResponse !== TX_REQUEST_NEXT_PART) {
```

**Review by graup — APPROVED** [review]

**Pull request merged**